packages/core/src/Box/Box.spec.js

```diff
@@ -46,30 +46,26 @@ describe('Box', () => {
     expect(json).toHaveStyleRule('color', theme.colors.blue)
   })
 
-  test('bg prop sets background color', () => {
-    const json = rendererCreateWithTheme(<Box bg='green' />).toJSON()
-    expect(json).toMatchSnapshot()
-    expect(json).toHaveStyleRule('background-color', theme.colors.green)
-  })
-
   test('boxShadowSize prop sets box-shadow', () => {
     const json = rendererCreateWithTheme(<Box boxShadowSize='sm' />).toJSON()
     expect(json).toMatchSnapshot()
     expect(json).toHaveStyleRule('box-shadow', theme.boxShadows[0])
   })
 
-  test('align prop triggers warning', () => {
-    console.error = jest.fn()
-
-    rendererCreateWithTheme(<Box align='center' />).toJSON()
+  describe('deprecated prop types', () => {
+    const consoleError = console.error
+    beforeEach(() => (console.error = jest.fn()))
+    afterEach(() => (console.error = consoleError))
 
-    expect(
-      console.error.mock.calls
-        .toString()
-        .indexOf(
-          'The Box `align` prop will deprecated. Please use Text instead.'
```

Comment:
This test was removed as the deprecated prop `align` has been replaced with `textAlign`, so there is no longer a warning thrown when passing `align`.